academic_observatory_api/client/api/observatory_api.py

```diff
@@ -0,0 +1,857 @@
+"""
+    academic-observatory-workflows API
+
+    The REST API for accessing data from academic-observatory-workflows.   # noqa: E501
+
+    The version of the OpenAPI document: 1.0.0
+    Contact: agent@observatory.academy
+    Generated by: https://openapi-generator.tech
+"""
+
+
+import re  # noqa: F401
+import sys  # noqa: F401
+
+from academic_observatory_api.client.api_client import ApiClient, Endpoint as _Endpoint
+from academic_observatory_api.client.model_utils import (  # noqa: F401
+    check_allowed_values,
+    check_validations,
+    date,
+    datetime,
+    file_type,
+    none_type,
+    validate_and_convert_types
+)
+from academic_observatory_api.client.model.pit_response import PitResponse
+from academic_observatory_api.client.model.query_response import QueryResponse
+
+
+class ObservatoryApi(object):
```

Comment:
I think it would be better if this was called `AcademicObservatoryApi`, I presume that something will need to be updated in the openapi.yaml file